specification/mediaservices/resource-manager/Microsoft.Media/preview/2020-02-01-preview/MediaGraphs.json

```diff
@@ -0,0 +1,866 @@
+        "description": "Stop a Media Graph in the Media Services account.",
+        "operationId": "MediaGraphs_Stop",
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "202": {
+            "description": "Accepted"
+          },
+          "default": {
+            "description": "Detailed error information.",
+            "schema": {
+              "$ref": "#/definitions/ApiError"
+            }
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/MediaServicesAccountName"
+          },
+          {
+            "name": "mediaGraphName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The Media Graph name."
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          }
+        ],
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Stop Media Graph": {
+            "$ref": "examples/media-graph-stop.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Media/mediaServices/{accountName}/mediaGraphs/{mediaGraphName}/operationResults/{operationId}": {
+      "get": {
+        "summary": "Get the operation result",
+        "description": "Get the operation result of a Media Graph in the Media Services account.",
+        "operationId": "OperationResults_Get",
+        "responses": {
+          "202": {
+            "description": "Accepted"
+          },
+          "204": {
+            "description": "NoContent"
+          },
+          "default": {
+            "description": "Detailed error information.",
+            "schema": {
+              "$ref": "#/definitions/ApiError"
+            }
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/MediaServicesAccountName"
+          },
+          {
+            "name": "mediaGraphName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The Media Graph name."
+          },
+          {
+            "name": "operationId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The operation ID"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          }
+        ],
+        "x-ms-examples": {
+          "Get the operation result": {
+            "$ref": "examples/operation-result-get.json"
+          }
+        }
+      }
+    }
+  },
+  "parameters": {

```

Comment:
nit: it would be nice to reference [common type](https://github.com/Azure/azure-rest-api-specs/blob/6daf2ac481925d891a427ba914ba46bff50dc7c5/specification/common-types/resource-management/v1/types.json#L438) with allof and add MediaServiceAccountName.

Reply:
Changed to using common types.